src/bake/DevServer.zig

```diff
@@ -166,6 +171,9 @@ deferred_request_pool: bun.HiveArray(DeferredRequest.Node, DeferredRequest.max_p
 /// UWS can handle closing the websocket connections themselves
 active_websocket_connections: std.AutoHashMapUnmanaged(*HmrSocket, void),
 
+relative_path_buf_lock: bun.DebugThreadLock,
+relative_path_buf: bun.PathBuffer,
```

Comment:
maybe make this `*bun.PathBuffer` and get this from `bun.PathBufferPool` instead?

Reply:
no. i am intentionally pre-allocating it with the already large `DevServer` object (already above 20kb)